test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -600,12 +610,22 @@ class AccountListPage {
     console.log(
       `Verify the number of accounts in the account menu is: ${expectedNumberOfAccounts}`,
     );
-    await this.driver.wait(async () => {
-      const internalAccounts = await this.driver.findElements(
-        this.accountListItem,
-      );
-      return internalAccounts.length === expectedNumberOfAccounts;
-    }, 20000);
+
+    await this.driver.waitForSelector(this.accountListItem);
+    await this.driver.wait(
+      async () => {
+        const internalAccounts = await this.driver.findElements(
+          this.accountListItem,
+        );
+        const isValid = internalAccounts.length === expectedNumberOfAccounts;
+        console.log(
+          `Number of accounts: ${internalAccounts.length} is equal to ${expectedNumberOfAccounts}? ${isValid}`,
+        );
+        return isValid;
+      },
+      20000,
+      true,
```

Comment:
could you check the change here? i don't think the change is necessary and correct here for function `check_numberOfAvailableAccounts`.

Reply:
this was a change proposed by Charlie but it was missing the return statement and that's why some unrelated tests were failing. I added this part and now they perfectly work